sdk/sbf/c/sbf.mk

```diff
@@ -202,9 +202,6 @@ $1: $2
 	@echo "[lld] $1 ($2)"
 	$(_@)mkdir -p $(dir $1)
 	$(_@)$(LLD) $(SBF_LLD_FLAGS) -o $1 $2 $(COMPILER_RT_DIR)/libcompiler_builtins-*.rlib
-ifeq (,$(wildcard $(subst .so,-keypair.json,$1)))
-	$(_@)solana-keygen new --no-passphrase --silent -o $(subst .so,-keypair.json,$1)
-endif
```

Comment:
seems we've reached to a consensus to remove this entirely: https://discord.com/channels/428295358100013066/560503042458517505/1082437459423543437